parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileWriter.java

```diff
@@ -988,6 +988,21 @@ void writeColumnChunk(ColumnDescriptor descriptor,
     endColumn();
   }
 
+  /**
+   * Overwrite the column total statistics. This special used when the column total statistics
+   * is known while all the page statistics are invalid, for example when rewriting the column.
+   *
+   * @param totalStatistics the column total statistics
+   * @throws IOException if there is an error while writing
+   */
+  public void endColumn(Statistics<?> totalStatistics) throws IOException {
```

Comment:
Exposing this to public is not a good idea. Other good suggestions are welcome.

Reply:
Since this is used only for invalid stats, is it better to add a `void invalidateStatistics()` which simply include line 988 and 990? Then you just need to call invalidateStatistics() and endColumn() in this case.

Reply:
This is used to set the column aggregated statistics. Do you mean to modify the `public void endColumn(Statistics<?> totalStatistics)` to `public void invalidateStatistics(Statistics<?> totalStatistics)`?

Reply:
Replaced with `invalidateStatistics `